### DISCUSSION TITLE
Detecting when a link button in a view has been clicked

### DISCUSSION THREAD

**Discussion by DMcP89** [Rapptz/discord.py#9217] [Q&A] [answered]
According to the [docs ](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.ui.Button.callback ) when a button component uses the url parameter there is no callback sent from the API. Does this mean it is impossible for the bot to detect when a link button has been pressed?

**Comment by Rapptz** [comment, marked as answer]
Yes it doesn't dispatch anything.
